### PR TITLE
Redis: add affinity parameter to Master StatefulSet.

### PR DESCRIPTION
**Pull request by desaintmartin**
Allow to setup pod/node affinity/anti-affinity for Redis Master.

### CODE REVIEW

File: stable/redis/values.yaml [hunk 1]
```diff
@@ -172,6 +172,8 @@ master:
   ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#taints-and-tolerations-beta-feature
   # nodeSelector: {"beta.kubernetes.io/arch": "amd64"}
   # tolerations: []
+  ## Redis Master pod/node affinity/anti-affinity
+  # affinity: {}
```
Comment (juan131) [inline stable/redis/values.yaml]: Leave it uncommented. See how it's implemented on "slave" section

File: stable/redis/templates/redis-master-statefulset.yaml [hunk 2]
```diff
@@ -40,6 +40,10 @@ spec:
         runAsUser: {{ .Values.master.securityContext.runAsUser }}
       {{- end }}
       serviceAccountName: "{{ template "redis.serviceAccountName" . }}"
+      {{- if .Values.master.affinity }}
```
Comment (juan131) [inline stable/redis/templates/redis-master-statefulset.yaml]: Use: `{{- with .Values.master.affinity }}`

**Review by juan131 — CHANGES_REQUESTED** [review]
Thanks for the contribution. Please bump Chart version on `Chart.yaml`

File: stable/redis/values.yaml (hunk 1 above)
Comment (desaintmartin) [reply stable/redis/values.yaml]: Fixed.

File: stable/redis/templates/redis-master-statefulset.yaml (hunk 2 above)
Comment (desaintmartin) [reply stable/redis/templates/redis-master-statefulset.yaml]: Fixed.

**Comment by desaintmartin** [comment]
Thanks for the suggestions. Fixed.

**Comment by juan131** [comment]
Great thanks!

**Pull request merged**